examples/dt/neutron/3cont_2comp-ipv6-all-geneve-dvr-stf-rally

```diff
@@ -0,0 +1,62 @@
+# Deployed Topology 3cont_2comp-ipv6-all-geneve-dvr-stf-rally
+
+## General information
+
+| Revision | Change              |  Date      |
+| -------: | :------------------ | :-----:    |
+|     v0.1 | Initial publication | 2024-01-16 |
+
+## Node topology
+
+| Node role                                  | bm/vm | amount |
+| ------------------------------------------ | ----- | ------ |
+| Openshift master/worker combo-node cluster | vm    | 3      |
+| Compute nodes                              | vm    | 2      |
+
+
+## Services, enabled features and configurations
+
+| Service   | configuration                | Lock-in coverage?  |
+| --------- | ---------------------------- | ------------------ |
+| Neutron   | ML2/OVN, Geneve              | Must have          |
+|           | ovn-router-flavors           |                    |
+| Glance    | default                      | Must have          |
+| Cinder    | default                      | Must have          |
+| Keystone  | default                      | Must have          |
+| TLS       | default                      | Must have          |
+| Compute   | DifferentHostFilter          | Must have          |
+
+### Additional information 
+
+#### Nova
+
+#### Cinder
+
+#### OVN
+add configuration of ipv6, OVNEncapTos=inherit
```

Comment:
IMO, no specific configuration we need to add to OVN for ipv6, as well as ipv4 mentioned for ovn in other jobs. Since ipv4/ipv6 here is the control plane setting it should be pretty transparent for neutron. I think we should just inherit some global ipv6 control plane config  (or ipv4 for other jobs)  that will be common for most jobs.

Reply:
A correction regarding ipv6. We do configure ovn-encap-ip when using ipv6. It is set to ipv6 address [this only could be achieved in 17.1 by setting infrared tripleo-overcloud parameter --network-protocol ipv6-all]
When using --network-protocol ipv6 in infrared, control-plane was using ipv6 addresses while ovn was still using ipv4 for tunnels. This was causing  test_dscp_inheritance_geneve to fail. 
The OVNEncapTos=inherit setting was also only just for this test. In case the value is not specified, the test is just skipped.

Reply:
Marking this as Resolved, because this is mapped to unified job uni-delta-ipv6 and in that job this config is added.